server/mlabns/util/production_check.py

```diff
@@ -0,0 +1,34 @@
+import re
+
+
+def is_production_site(site_name):
+    """Determines if the given site name matches the production site schema
+
+    Args:
+        site_name: Name of site to check (e.g. "lga01")
+
+    Returns:
+        True if the site name matches the schema of a production site.
+    """
+    return re.match('^[a-z]{3}\d{2}$', site_name, re.IGNORECASE) != None
+
+
+def is_production_slice(slice_fqdn):
+    """Determines if the given FQDN matches the schema for a production slice
+
+    Args:
+        slice_fqdn: Slice FQDN to check (e.g. "lga01")
```

Comment:
Tiny detail, since it's only a comment but should the example list an a FQDN instead of a site name?